datajoint/autopopulate.py

```diff
@@ -323,9 +323,9 @@ def progress(self, *restrictions, display=True):
         total = len(todo)
         remaining = len(todo - self.target)
         if display:
```

Comment:
Then does it even make sense to have this `display` argument?

Reply:
Let's revisit in another PR. `display_progress` might need to be looking into as well.

The downside of removing this option is that using the `log_level` to control the display means the user is enabling a lot more logging than desired.